src/Components/Courses/NavigationMenuItem.tsx

```diff
@@ -0,0 +1,13 @@
+const NavigationMenuItem: React.FC<any> = ({label}) => {
```

Comment:
1. Set the name like `CourseCard` and move to the `src/components/CourseCard`
2. Files in the folder: `index.tsx`, `style.ts`, `types.ts`
3. Rewrite to class component
4. Set interface for props in the file `types.ts` for `Card`